packages/element-library/src/video/display.js

```diff
@@ -46,6 +46,7 @@ function VideoDisplay({
     focalX,
     focalY,
     loop,
+    volume = 1,
```

Comment:
```suggestion
    volume,
```

As per other discussion, this is basically what I meant. For a browser, `1` is the default volume anyway, so we don't need to specify it.